### PR TITLE
Actually finish flipping the Y axis

### PR DESCRIPTION
**Pull request by astronouth7303**
So that easy little y-flip PR I did left a bunch of stuff broken.

Fix that.

### CODE REVIEW

File: tests/test_camera.py [hunk 1]
```diff
@@ -42,8 +42,16 @@ def test_camera_translate_to_viewport():
...
+    assert cam.translate_to_viewport(Vector(5, 5)) == Vector(400, 300)
+    assert cam.translate_to_viewport(Vector(7, 4)) == Vector(560, 380)
+
+
+def test_camera_translate_to_viewport_2():
+    cam = Camera(viewport=(0, 0, 800, 600), pixel_ratio=1)
+    assert cam.position == Vector(0, 0)
+    assert cam.translate_to_viewport(Vector(0, 0)) == Vector(400, 300)
+    assert cam.translate_to_viewport(Vector(100, 100)) == Vector(500, 200)
+    assert cam.translate_to_viewport(Vector(-150, -500)) == Vector(250, 800)
```
Comment (astronouth7303) [inline tests/test_camera.py]: Do we want to parameterize these tests? It seems like we have a lot of tests that are the same basic format?

Comment (pathunstrom) [reply tests/test_camera.py]: I'll do that when I find a spare moment.

**Comment by pathunstrom** [comment]
This looks good, rerunning the failed tests, will merge when it's done.

**Comment by nbraud** [comment]
@pathunstrom FYI, you can always invoke `bors r+` and it will happily wait for tests to be done (or timeout after 3h) before merging.  It also does the test on a merge with the target branch, so manually merging `master` into this wasn't necessary.  :3

bors r=pathunstrom

**Comment by pathunstrom** [comment]
I mean, I intentionally wanted to make sure it was working before I walked away from it.

**Comment by nbraud** [comment]
'k, then I misunderstood the comment (but I did check the CI log before kicking off the merge)

**Comment by pathunstrom** [comment]
Then all's good. Just explaining my thought process, no worries.

**Pull request merged**